velox/exec/WindowFunction.h

```diff
@@ -31,6 +31,23 @@ struct WindowFunctionArg {
   std::optional<const column_index_t> index;
 };
 
+/// The scope for calculating the window function. kRows indicates that the
+/// calculation begins as soon as rows are available within a single partition,
+/// without waiting for all data in the partition to be ready. kPartition
+/// indicates that the calculation begins only when all rows in a partition are
+/// ready.
+enum class Scope {
+  kPartition,
+  kRows,
+};
+
+/// Indicates whether the function is for an aggregate used as a window
```

Comment:
typos and inaccurate description; the description covers 'isAggregateWindow' field, but not 'scope'.

Reply:
@mbasmanova Yes. Refined.